src/components/elements/side-tab.js

```diff
@@ -61,12 +94,22 @@ function useTabs(initial_active_tab = '', has_hash_routing) {
         if (has_hash_routing) navigate(`#${tab}`)
     }
 
-    return [active_tab, setTab]
+    return [active_tab, setTab, last_active_tab, setLastActiveTab]
```

Comment:
you mean, previous tab?

Reply:
yes, but I'm gonna change the name to be more understandable.